TarponSimulator2017/Core/FishingRod.cs

```diff
@@ -14,17 +16,29 @@ public class FishingRod : GameObject, IUpdatable
 		/// <value>The fishing float.</value>
 		public FishingFloat FishingFloat { get; private set; }
 
+		/// <summary>
+		/// Gets the list of fishes.
+		/// </summary>
+		/// <value>The list of fishes.</value>
+		public IList<Fish> ListOfFishes { get; private set; }
```

Comment:
Another list of fishes! The 3rd :smile: 
I mean, I think we should express what they are for.